activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+*   PostgreSQL adapter method `enable_extension` now allows to add a `schema:` parameter
+    to be given if the extension must be installed on another schema.
+
+    This patch tries to solve Heroku's new [PostgreSQL extension policy](https://devcenter.heroku.com/changelog-items/2446) while keeping the migration and schema code idiomatic.
```

Comment:
I don't think this part is necessary. Could you instead include a code example (and update the first line based on the new structure)?

Reply:
Yes, I had to commit against my will, the Codespace just died and I could run the test, so I had to commit and now I'm starting the Codespace back again, I'll finish updating ASAP.